examples/deno/README.md

```diff
@@ -0,0 +1,50 @@
+# Welcome to [Astro](https://astro.build)
+
+[![Open in StackBlitz](https://developer.stackblitz.com/img/open_in_stackblitz.svg)](https://stackblitz.com/github/withastro/astro/tree/latest/examples/basics)
+
+> 🧑‍🚀 **Seasoned astronaut?** Delete this file. Have fun!
+
+![basics](https://user-images.githubusercontent.com/4677417/186188965-73453154-fdec-4d6b-9c34-cb35c248ae5b.png)
+
+
+## 🚀 Project Structure
+
+Inside of your Astro project, you'll see the following folders and files:
```

Comment:
Just wanted to check that this tree diagram of files does accurately represent what's included in the starter? I notice this mentions a `Card.astro` in a `components` folder, but the layout of files I see changed here in this PR instead includes a `Layout.astro` inside that folder (and, does not include a `layouts` folder).

So, no issues with the README from a Docs perspective other than I just wanted to check on the files we were claiming were included! When that is confirmed, LGTM!

Reply:
Hey @sarah11918 , you were right! I updated the README!

Reply:
Thanks @ElianCodes! If the `tsconfig.json` is included in the starter, I'd put it there, too, since that's a key Astro file.

I'm torn on the `sandbox.config.json` since it's not strictly-speaking necessary. I think it's fine to leave that one out?

Reply:
I think I would also leave the `sandbox.config.json` out! I've included the `tsconfig.json`!